metrics/ci_integrations/docs/diagrams/ci_integrations_config_validator_class_diagram.puml

```diff
@@ -79,10 +79,21 @@
         }
     }
 
-     package client.cool_integration as client_cool_integration {
+    package client.cool_integration as client_cool_integration {
         class CoolIntegrationClient {}
     }
 
+    package cli.command {
+        class ValidateCommand {
```

Comment:
Would it be great to specify that this command extends the `CiIntegrationCommand` abstract class?